rust/main/agents/relayer/src/msg/metadata/ccip_read/mod.rs

```diff
@@ -167,7 +167,6 @@ impl CcipReadIsmMetadataBuilder {
 
 #[async_trait]
 impl MetadataBuilder for CcipReadIsmMetadataBuilder {
-    #[instrument(err, skip(self, message, _params))]
```

Comment:
could we add all the `build` ones back too? as these are just for `err`s and not `ret`s, they likely don't contribute much too